pkg/estimator/server/framework/plugins/noderesource/noderesource.go

```diff
@@ -134,11 +139,10 @@ func (pl *nodeResourceEstimator) EstimateComponents(_ context.Context, snapshot
 	return sets, framework.NewResult(framework.Success)
 }
 
-// getNodeRestResource calculates the remaining available resources for each node in the cluster.
-// It clones each node and subtracts the already requested resources and existing pod count
-// to determine how much capacity is left for new workloads.
-// Returns a slice of NodeInfo with updated allocatable resources representing available capacity.
-func getNodeRestResource(snapshot *schedcache.Snapshot) ([]*schedulerframework.NodeInfo, error) {
+// getNodeInfos retrieves and prepares the list of node information from the snapshot.
+// It clones each node's info and adjusts the allocatable resources by subtracting the requested resources.
+// So that the returned node infos reflect the actual available resources for scheduling.
+func getNodeInfos(snapshot *schedcache.Snapshot) ([]*schedulerframework.NodeInfo, error) {
```

Comment:
The function name is a bit common, and does not reflect that it computes available resources. 
Another option would be: `getNodesAvailableResources`